### ISSUE TITLE
Add test for the ability to create resources in YAML for resources without forms

### ISSUE THREAD

**Issue by catherineluse** [rancher/dashboard#5092]
Vince mentioned this was a regression. There has already been a fix for this bug, but a test should also be added to prevent another regression. https://github.com/rancher/dashboard/issues/5034

**Comment by cnotv** [comment]
It's the test intended to aim to this button?

![Screenshot 2022-07-28 at 11 49 32](https://user-images.githubusercontent.com/5009481/181477535-1cf15d6f-f935-4e01-bb12-31808af034cc.png)

**Comment by catherineluse** [comment]
No, the request is for testing the UX for resources that don't have any forms for them in the UI. This is usually either because they are obscure, or not recommended to be manually updated, or because they are a custom resource that only some users would have. For example, Leases:
<img width="1413" alt="Screen Shot 2022-07-28 at 9 02 18 PM" src="https://user-images.githubusercontent.com/20599230/181680595-38f3f52b-e9e3-438b-ac71-f9c2f1915ab0.png">

You would basically need to:

- confirm that the **Create from YAML** button exists on the list view for the resource
- confirm that you can successfully create the resource from YAML
<img width="1422" alt="Screen Shot 2022-07-28 at 9 02 23 PM" src="https://user-images.githubusercontent.com/20599230/181680613-1b770a28-eb3c-4d6f-bbb8-b2d393337eeb.png">

**Comment by cnotv** [comment]
I would rather stick to the second case, as the first one is unit test based on client files.

**Comment by catherineluse** [comment]
The error that caused this issue to be opened was when the Create from Yaml button just didn't show up.

**Comment by cnotv** [comment]
Yes, but a you can see it's something which can be tested inside this component, given the PR
https://github.com/rancher/dashboard/pull/4997/files
https://github.com/jordojordo/dashboard/blame/d9245c9962c0e44264fe929013c789379de2454c/components/ResourceList/Masthead.vue#L101
